package.json

```diff
@@ -55,6 +54,7 @@
     "eslint-config-uber-jsx": "^3.0.0",
     "eslint-plugin-react": "~6.7.0",
     "gl": "^4.0.3",
+    "immutable": "*",
```

Comment:
Some of our tests/examples are using immutable still? Would it be possible not to use `*`?